perk-store-services/src/main/java/org/exoplatform/perkstore/service/PerkStoreService.java

```diff
@@ -307,6 +307,8 @@ public Product saveProduct(Product product, String username) throws Exception {
       }
     }
     List<ProductOrder> orders = null;
+    Boolean isPerkStoreManager = isPerkStoreManager(username);
+    filter.setMyOrders(!isPerkStoreManager);
```

Comment:
What if the user isn't manager of perk store but manager of a product, will he get the list of orders of his product ?

Reply:
yes since he became the receiver of the order

Reply:
ok, we will need to add unit tests to reflect all use cases